test/performance/LatencyTestPublisher.cpp

```diff
@@ -270,6 +270,9 @@ bool LatencyTestPublisher::init(int n_sub, int n_sam, bool reliable, uint32_t pi
     PubDataparam.topic.topicName = pt.str();
     PubDataparam.times.heartbeatPeriod.seconds = 0;
     PubDataparam.times.heartbeatPeriod.fraction = 4294967 * 100;
+    PubDataparam.qos.m_liveliness.lease_duration = c_TimeInfinite;
```

Comment:
Why this new configuration?

Reply:
We added these configuration fields to accelerate the TCP discovery process, with the default parameters the discovery could take around 40 seconds because of the TCP negotiation.

Reply:
After checking how these parameters could affect the performance... we have removed them.